word_guess.rb

```diff
@@ -0,0 +1,198 @@
+require 'colorize'
+
+class WordGuess
+	attr_accessor :guesses_wrong, :win
+
+	def initialize
+    @correct_word = %w(H A N G M A N)
+    @guessing_array = %w(_ _ _ _ _ _ _)
+		@guesses_wrong = 0
+    @wrong_letters = []
+    @counter = "poop"
+		@win = false
+	end
+
+	def get_user_input
+		# gets user input
+    # poop consistency!
+    choice = "poop"
+    until choice == "WORD" || choice == "LETTER"
+      puts "Would you like to guess a word or a letter?"
+      choice = gets.chomp.upcase
+      if choice == "WORD"
+        guess_word(choice)
+      elsif choice == "LETTER"
+        guess_letter(choice)
+      else
+        puts "Incorrect input."
+      end
+    end
+	end
+
+  def guess_word(choice)
+    print "What is your guess? "
+    word_guess = gets.chomp.upcase
+      while word_guess.empty?
+        puts "Sorry, entries can not be blank!"
+        print "What is your guess? "
+        word_guess = gets.chomp.upcase
+      end
+    # splits the word into an array of letters
+		word_guess = word_guess.split(//)
+		if word_guess == @correct_word
+			show_win
+    else
+      puts "Sorry!"
+      show_word
+      @guesses_wrong += 1
+			show_strikes
+		end
+  end
+
+  def guess_letter(choice)
+  	print "What is your guess? "
+  	letter_guess = gets.chomp.upcase
+    while letter_guess.empty? || letter_guess.length > 1 || @wrong_letters.include?(letter_guess)
+      puts "Sorry, that entry is incorrect!"
+      puts "Incorrect entries include empty input,"
+      puts "multiple letter entry and duplicate entry."
+      print "What is your guess? "
+      letter_guess = gets.chomp.upcase
+    end
+  	# calls method to check letter against correct word array
+    check_letter(letter_guess)
+  	# counter allows a return of letters that are the same character
+    # return keyword would exit before reaching the second same letter
+    if @counter == "on"
+  		puts "That letter was right! Good job!"
+  		@counter = "off"
+  		show_strikes
+  		show_word
+  		show_wrong_letters
+      if !@guessing_array.include?("_")
+        show_win
+      end
```

Comment:
I think that this `end` will prevent the `else` on the following line from executing. I think the `end` should be present after line 76 instead.